examples/resources/infisical_cert_manager_certificate_profile/resource.tf

```diff
@@ -0,0 +1,193 @@
+  }
+}
+
+# API enrollment profile for web server certificates
+resource "infisical_cert_manager_certificate_profile" "web_server_api" {
+  project_slug            = infisical_project.pki.slug
+  ca_id                   = infisical_cert_manager_internal_ca_intermediate.issuing.id
+  certificate_template_id = infisical_cert_manager_certificate_template.web_server.id
+
+  name            = "web-server-api"
+  description     = "API enrollment for web server certificates"
+  enrollment_type = "api"
+  issuer_type     = "ca"
+
+  api_config {
+    auto_renew        = true
+    renew_before_days = 7
+  }
+}
+
+# EST enrollment profile for web server certificates
+resource "infisical_cert_manager_certificate_profile" "web_server_est" {
+  project_slug            = infisical_project.pki.slug
+  ca_id                   = infisical_cert_manager_internal_ca_intermediate.issuing.id
+  certificate_template_id = infisical_cert_manager_certificate_template.web_server.id
+
+  name            = "web-server-est"
+  description     = "EST enrollment for web server certificates"
+  enrollment_type = "est"
+  issuer_type     = "ca"
+
+  est_config {
+    passphrase                      = var.est_passphrase
+    disable_bootstrap_ca_validation = false
+    ca_chain                        = var.encrypted_ca_chain
+  }
+}
+
+# Self-signed profile for development
+resource "infisical_cert_manager_certificate_profile" "self_signed_dev" {
+  project_slug            = infisical_project.pki.slug
+  certificate_template_id = infisical_cert_manager_certificate_template.web_server.id
+
+  name            = "self-signed-dev"
+  description     = "Self-signed certificates for development"
+  enrollment_type = "api"
+  issuer_type     = "self-signed"
+
+  api_config {
+    auto_renew        = false
+    renew_before_days = 7
+  }
+}
+
+# ACME profile
+resource "infisical_cert_manager_certificate_profile" "acme_profile" {
+  project_slug            = infisical_project.pki.slug
+  ca_id                   = var.acme_ca_id
+  certificate_template_id = infisical_cert_manager_certificate_template.web_server.id
+
+  name            = "acme-letsencrypt"
+  description     = "Let's Encrypt ACME certificates"
+  enrollment_type = "acme"
+  issuer_type     = "ca"
+}
+
+# ADCS profile (requires external ADCS CA to be configured)
+resource "infisical_cert_manager_certificate_profile" "adcs_profile" {
+  project_slug            = infisical_project.pki.slug
+  ca_id                   = var.adcs_ca_id # Reference to existing ADCS CA
+  certificate_template_id = infisical_cert_manager_certificate_template.web_server.id
+
+  name            = "adcs-corporate"
+  description     = "Corporate ADCS certificates"
+  enrollment_type = "api"
+  issuer_type     = "ca"
+
+  api_config {
+    auto_renew        = true
+    renew_before_days = 14
+  }
+
+  external_configs {
+    template = "WebServerTemplate" # ADCS template name
+  }
+}
+
```

Comment:
IGNORE PLACEMENT OF COMMENT

There seems to be an infinite-drift with the passphrase field?
<img width="899" height="827" alt="image" src="https://github.com/user-attachments/assets/60939f99-f98a-491a-aeae-be6f70dab09a" />